csharp-advanced-topics/EventsDrivenArchitectureInCsharp/LightControlService/LightSwitchEventSubscriber.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Events;
+using MassTransit;
+
+namespace LightControlService;
+
+public class LightSwitchEventSubscriber : IConsumer<LightSwitchEvent>
+{
+    public async Task Consume(ConsumeContext<LightSwitchEvent> context)
+    {
+        var lightEvent = context.Message;
+        bool success = await ControlLightsAsync(lightEvent);
```

Comment:
Use `var` where possible, also since we are using this in an `if` statement later on, let's modify the naming here to be in a form of a question, eg. `isSuccessful`